package.json

```diff
@@ -24,7 +24,13 @@
     "prepare-release": "./prepare-release.sh",
     "release": "./release.sh",
     "prune-arc-build": "node ./pruneArcBuild.js",
-    "postinstall": "npm run prune-arc-build"
+    "postinstall": "npm run prune-arc-build",
+    "verify.initialize": "npm explore @daostack/arc -- npm install openzeppelin-solidity @daostack/infra --no-save && npm explore @daostack/infra -- npm install openzeppelin-solidity --no-save && npm explore @daostack/arc-hive -- npm install openzeppelin-solidity --no-save",
+    "verify.build": "node node_modules/typescript/bin/tsc verify.ts -sourceMap",
+    "verify.help": "node verify.js -h",
```

Comment:
I think this is redundant, everyone knows how to use `-h` or `--help`

Reply:
they don't know to how to run `node verify.js -h`

Reply:
`npm run verify -- -h` or `yarn verify -h`

Reply:
i don't see any reason not to have this helpful shortcut.  risk-wise it is trivial